extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/AstroHandlerFactory.java

```diff
@@ -29,20 +29,23 @@
  * @author Gerhard Riegler - Initial contribution
  */
 public class AstroHandlerFactory extends BaseThingHandlerFactory {
-    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Sets.union(SunHandler.SUPPORTED_THING_TYPES,
+
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Sets.union(SunHandler.SUPPORTED_THING_TYPES,
             MoonHandler.SUPPORTED_THING_TYPES);
-    private static final Map<String, AstroThingHandler> astroThingHandlers = new HashMap<String, AstroThingHandler>();
+    private static final Map<String, AstroThingHandler> astroThingHandlers = Maps.newHashMap();
```

Comment:
I agree. Just to be concise from the source code perspective, I think we can use this. Apart from this, we are also using Preconditions from Guava already so I think we could use this as well.